contrib/custom-image-kubelet/genconfig.py

```diff
@@ -167,6 +168,8 @@ def write_ecr_credentials(release, prefix):
 
     for registry in registries:
         parts = registry.split('.')
+        if len(parts) < 4:
```

Comment:
What does the number 4 represent? Can this be made a variable with a descriptive name?

Reply:
Changed out the split for a regex: https://github.com/rancher/rke2/pull/554/commits/7e1226d968eecef6adb93fcc75e65e7a8e3b3194#diff-e81b4d596775d35ffea04790f3aca8eb5c7ef803b51e6a0860fd66fa7c65c71aR30